File: help/pt_pt/exercicios_da_ficha_3/Ficha03_010/Ficha03_010.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_010 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Valor de X\t\t: " );
        double x = Double.valueOf(keyb.nextLine());
        double logX = Math.log10( x ) ;
        System.out.print("Log10(X) \t\t= " + logX );
        double pow10 = Math.pow( 10 , logX ) ;
        System.out.print("\nPow10(Log10(X)) \t= " + pow10 );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:30

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_010/Ficha03_010.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha03_010 {

    //Main Function
    static void Main() {
        Console.Write("Valor de X\t\t: " );
        double x = double.Parse(Console.ReadLine());
        double logX = Math.Log10( x ) ;
        Console.Write("Log10(X) \t\t= " + logX );
        double pow10 = Math.Pow( 10 , logX ) ;
        Console.Write("\nPow10(Log10(X)) \t= " + pow10 );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:30
